src/main/java/com/google/sps/utils/Game.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.utils;
+import com.google.appengine.api.datastore.Entity;
+import java.lang.IllegalArgumentException;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+ 
+public final class Game {
+    DatastoreService datastore;
+    private Entity entity;
+ 
+    public Game() {
+        datastore = DatastoreServiceFactory.getDatastoreService();
+        entity = new Entity("Game");
+    }
+ 
+    public void setGameId(String gameId) {
+        entity.setProperty("gameId", gameId);
+        datastore.put(entity);
```

Comment:
You can take out the datastore.put() calls (and datastore in general) and we'll just add it to datastore where the user object is called, that way it requires less calls to this

Reply:
done!